config/alphagov.yml

```
@@ -1,8 +1,40 @@
 # Please insert new items alphabetically.
 
-FUN-workstream:
+find-and-view-tech:
+  channel:
+    "#find-and-view-tech"
+  members:
+    - "1pretz1"
+    - AgaDufrat
+    - andyhd
+    - "chao-xian"
+    - "chris-gds"
+    - edwardkerry
+    - hannako
+    - hannalaakso
+    - "jon-kirwan"
+    - "KludgeKML"
+    - kashifatcha
+    - martinjjones
+    - matthillco
+    - maxgds
+    - patrickpatrickpatrick
+    - Tetrino
+  compact: true
+  exclude_titles:
+    - "[DO NOT MERGE]"
+    - "[PROTOTYPE]"
+    - WIP
+    - "[WIP]"
+  quotes:
+    - "Be Excellent To Each Other!"
+    - ":smiling-batman:"
+    - "Don't be working if you're ill!"
+    - ":green_heart:"
+
+fun-workstream-gds-community:
   channel:
-    "#sealtesting"
+    "#govuk"
```

Should this be "#govuk" or "#gds-community"? Just checking because the both fun-workstream-gds-community and fun-workstream-govuk have the channel "#govuk" and wasn't sure if it was intentional.

---

Ahh, thanks for spotting this :) 

Yes this should be for #gds-community 

is there also one for fun-workstream-govuk:? that should be for #govuk

---

I think the channel name will need updating then to
```suggestion
    "#gds-community"
```

and yes there is other one for fun-workstream-govuk defined here: https://github.com/alphagov/seal/blob/f9bfb42b71284456458bb44b49c1ce0b3c3f2f3d/config/alphagov.yml#L70-L102